tests/integration_tests/functional/test_mmds.py

```diff
@@ -693,10 +712,10 @@ def test_patch_dos_scenario(test_microvm_with_api):
 
 def test_mmds_snapshot(bin_cloner_path):
     """
-    Exercise tokens' behavior with snapshots.
+    Exercise MMDS behavior with snapshots.
 
-    Ensures that valid tokens created on a base microVM
-    are not accepted on the clone VM.
+    Ensures that MMDS V2 behavior is not affected by taking a snapshot
+    and that MMDS V2 is not available after snapshot load.
```

Comment:
this calls for a changelog update

Reply:
Not sure about a changelog update, since this snapshot limitation can't really be classified as `added`, `changed` or `fixed`. This will be mentioned in the mmds-related docs (upcoming changes in the documentation PR in progress). Would you be fine with this?

Reply:
Hmm I'd add this limitation to this changelog line:

```
New optional version field to PUT requests towards /mmds/config to configure MMDS version. Accepted values are V1 and V2 and default is V1.
```

I'd add: `Known limitation: V2 does not currently work after snapshot load.`

I'm keen on having this in the changelog since I believe there may be many users that don't regularly look into the design docs.

Reply:
This sounds good! I updated the CHANGELOG.md with your suggestion.